### PR TITLE
My code review

### PR DESCRIPTION
**Pull request by Ogaga01**
Added the files to be reviewed

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FCC: Survey Forms</title>
+    <link rel="stylesheet" href="./styles/style.css" />
+  </head>
+  <body>
+    <main>
+      <h1>freeCodeCamp Survey Form</h1>
+      <p id="para">Thank you for taking time to help us improve the platform</p>
```
Comment (Ogaga01) [inline index.html:13]: I think this should be a section different from the forms section. try putting them into different sections for better readablity.

**Review by Ogaga01 — COMMENTED** [review]
❌ Linters not implemented
❌ GitHub flow not implemented

Good job on adding the HTML and CSS code
.
# changes required ♻️ 

Please check my comment below and make all required changes

File: index.html (hunk 1 above)
Comment (sja-thedude) [reply index.html:13]: Good job 👍 in identifying the proper use of semantic tags.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
Hi @Ogaga01, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good in identifying the use of semantic tags.
- Good 👍 identifying that linters are not added
- Good 👍 following gitflow.

## Required Changes ♻️
- Kindly mention: Readme file should be professional as it is the overview of the project.
- Remember to mention: Pull request description should be descriptive.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ogaga01 — COMMENTED** [review]
❌ linters not properly implemented.
❌ GitHub flow not properly implemented
✔️ Professional commit message

Good job so far. 

# Required Changes ♻️ 

1. Readme file should be professional as it is the overview of the project.
2. Pull request description should be descriptive

**Review by Mre55 — APPROVED** [review]
**Hi @Ogaga01m,**

# Status: APPROVED  💪

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

Alt Text

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**